flottform/server/src/database.ts

```diff
@@ -25,8 +26,46 @@ function createRandomEndpointId(): string {
 
 class FlottformDatabase {
 	private map = new Map<EndpointId, EndpointInfo>();
+	private cleanupTimeoutId: NodeJS.Timeout | null = null;
+	private cleanupPeriod: number;
+	private entryTTL: number; // Time-to-Live for each entry
```

Comment:
How about `entryTimeoutInMs` to make it clear we're expecting milliseconds here? Also, `TTL` gets explained in a comma, so I'd suggest to get rid of it by not using an abbreviation. Abbreviations in general, I'd favor `entryTtl`, so if we end up having another abbreviation before or after this one, it can stay consistent (`HttpId` vs `HTTPID` or something along those lines...)